### PR TITLE
Updating all ISC22 to SC22.

### PR DESCRIPTION
**Pull request by t-brown**
- changed all references from ISC22 to SC22.
- update the date to now use a global param (and to Nov 14th).
- changed all regions to be us-east-2.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/01-hpc-overview/00-agenda.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ draft: false
 weight: 10
 ---
 
-You will find here the agenda for the tutorial. An update will be made on May 29th 2022 with references to the different workshops.
+You will find here the agenda for the tutorial. An update will be made on {{< param tutorialDate >}} with references to the different workshops.
```
Comment (mhuguesaws) [inline content/01-hpc-overview/00-agenda.md:8]: Nice!

File: content/01-hpc-overview/01-updates.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Frequently asked questions will be addressed on this page during the tutorial. D
 
 #### Accounts access
 
-Sandbox are available on May 29th 2022 for the duration of the tutorial. If you would like to run through the labs at a later stage on your own, with your company or institution, please contact us at isc22tutorial@amazon.com so we can follow-up with you.
+Sandbox are available on {{< param tutorialDate >}} for the duration of the tutorial. If you would like to run through the labs at a later stage on your own, with your company or institution, please contact us at sc22tutorial@amazon.com so we can follow-up with you.
```
Comment (mhuguesaws) [inline content/01-hpc-overview/01-updates.md:14]: @t-brown could you setup a variable as well for the email address ?

File: content/01-hpc-overview/00-agenda.md [hunk 3]
```diff
@@ -5,9 +5,9 @@ draft: false
...
 
-### Sunday May 29th 2022
+### {{< param tutorialDate >}}
```
Comment (mhuguesaws) [inline content/01-hpc-overview/00-agenda.md:10]: Please update agenda.

File: content/04-container-parallelcluster/02-update-PC.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ Click next twice to advance to the **Queues section**, here we're going to add a
 Next add in a script that installs Docker and Singularity on the Compute Nodes.
```
Comment (mhuguesaws) [inline content/04-container-parallelcluster/02-update-PC.md:44]: could we delete this lab please?

File: content/06-batch-automation/03-setup-batch-ce-jq.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ Run the following commands on your Cloud9 terminal
 1. Copy the Cloudformation template which will be used to create the AWS Batch Compute Environment and Job Queue.
 
 ```bash
```
Comment (mhuguesaws) [inline content/06-batch-automation/03-setup-batch-ce-jq.md:18]: Delete the lab

File: content/_index.md [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
-# ISC22 Tutorial
+# SC22 Tutorial
 
-Welcome to the **Best Practices for HPC in the Cloud** tutorial at ISC22 that will be delivered on May 29th 2022. We are excited to virtually meet you, and look forward to answering your questions!
-The website will be maintained after ISC22, labs will also be available on https://www.hpcworkshops.com within a few weeks after the conference.
+Welcome to the **Best Practices for HPC in the Cloud** tutorial at SC22 that will be delivered on {{< param tutorialDate >}} . We are excited to meet you, and look forward to answering your questions!
+The website will be maintained after SC22, labs will also be available on https://www.hpcworkshops.com within a few weeks after the conference.
 
-You can download the slide on [https://isc22-slides.s3.eu-west-1.amazonaws.com/ISC22_Best_Practices_HPC_in_The_Cloud.pdf](https://isc22-slides.s3.eu-west-1.amazonaws.com/ISC22_Best_Practices_HPC_in_The_Cloud.pdf)
+You can download the slide on [https://sc22-hpc-slides.s3.us-east-1.amazonaws.com/SC22_Best_Practices_HPC_in_The_Cloud.pdf](https://sc22-hpc-slides.s3.us-east-1.amazonaws.com/SC22_Best_Practices_HPC_in_The_Cloud.pdf)
```
Comment (mhuguesaws) [inline content/_index.md:12]: You can remove that line. The conference will provide the slides.

**Review by mhuguesaws — COMMENTED** [review]
Left comments

File: content/01-hpc-overview/01-updates.md (hunk 2 above)
Comment (t-brown) [reply content/01-hpc-overview/01-updates.md:14]: I tried, but it wouldn't become a clickable `mailto` link. Will keep digging.

Comment (t-brown) [reply content/01-hpc-overview/01-updates.md]: Managed to do it.

**Pull request closed without merging**